metrics/registry.go

```diff
@@ -0,0 +1,11 @@
+package metrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+var PelicanRegistryFederationNamespaces = promauto.NewGaugeVec(prometheus.GaugeOpts{
+	Name: "pelican_registry_federation_namespaces",
+	Help: "The number of namespaces in a federation",
```

Comment:
Let's try to be more clear about what "namespaces" means here. 
```suggestion
	Help: "The number of federation namespace associated with a public key, excluding server namespaces, in the registry.",
```